.golangci.yml

```diff
@@ -1,4 +1,4 @@
-linters-settings:
+xlinters-settings:
```

Comment:
Is this change intentional?

Reply:
no, I think that golangci did it somehow...